app.rb

```diff
@@ -0,0 +1,120 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  attr_accessor :people, :books, :rentals
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'No books available.'
+    else
+      puts 'All Books:'
+      @books.each do |book|
+        puts "Title: #{book.title} || Author: #{book.author}"
+      end
+    end
+  end
+
+  def list_all_people
+    if @people.empty?
+      puts 'No people available.'
+    else
+      puts 'All People:'
+      @people.each do |person|
+        puts "ID: #{person.id} || Name: #{person.name} || Type: #{person.class}"
```

Comment:
- [ ] When displaying the list of people, please make sure to include the `age` of the person to match the sample video. 😎️

![image](https://github.com/christianonoh/school-library/assets/17243325/0739955a-3837-4780-9af6-e371bb1d4d58)